js/jtline.js

```diff
@@ -63,17 +63,21 @@
                 var timelineTotWidth = setTimelineWidth(timelineComponents, options.eventsMinDistance);
                 //the timeline has been initialize - show it
                 timeline.addClass('loaded');
-
-                //detect click on the next arrow
-                timelineComponents['timelineNavigation'].on('click', '.next', function (event) {
-                    event.preventDefault();
-                    updateSlide(timelineComponents, timelineTotWidth, 'next');
-                });
-                //detect click on the prev arrow
-                timelineComponents['timelineNavigation'].on('click', '.prev', function (event) {
-                    event.preventDefault();
-                    updateSlide(timelineComponents, timelineTotWidth, 'prev');
-                });
+                if(screen.width < screenSize){
```

Comment:
This isn't working properly. The buttons are inactive even in smaller screen widths. Have you tested this properly ?

Reply:
Yes tested...It worked could you tell me how you're testing for different screens

Reply:
Okay It is working fixing. The fix is somewhere only.

Reply:
Also what is the smaller screen width upper limit we need to be setting if the width greater than 1008 it won't work